Code/Editor/EditorFramework/PropertyGrid/Implementation/ExposedParametersPropertyWidget.cpp

```diff
@@ -250,6 +250,8 @@ void ezQtExposedParameterPropertyWidget::InternalSetValue(const ezVariant& value
           m_pWidget->setParent(this);
           m_pLayout->addWidget(m_pWidget);
           m_pWidget->Init(m_pGrid, m_pObjectAccessor, type, prop);
+
+          UpdateTypeListSelection(commonType);
```

Comment:
why is this needed here ?

Reply:
The exposed parameter widget derives from variant widget which has been refactored. This call is needed so the exposed parameter widget has the correct variant type preselected in its type dropdown.